### PR TITLE
Added Data Source debugger

### PR DESCRIPTION
**Pull request by kkrampa**
@NoahCarnahan @nickpell 
Extension to Cory's expression debugger. What do you think about?

data-source

### CODE REVIEW

**Comment by emord** [comment]
Awesome! I've really wanted this before

File: corehq/apps/userreports/views.py [hunk 1]
```diff
@@ -962,6 +968,34 @@ def evaluate_expression(request, domain):
...
+@toggles.USER_CONFIGURABLE_REPORTS.required_decorator()
+def evaluate_data_source(request, domain):
+    data_source_id = request.POST['data_source']
+    docs_id = request.POST['docs_id']
+    try:
+        data_source = get_datasource_config(data_source_id, domain)[0]
+        docs_id = [doc_id.strip() for doc_id in docs_id.split(',')]
+        document_store = get_document_store(domain, data_source.referenced_doc_type)
+        rows = []
+        for doc in document_store.iter_documents(docs_id):
```
Comment (snopoke) [inline corehq/apps/userreports/views.py]: This should validate that the document is from the same domain as the request

**Review by snopoke — COMMENTED** [review]
Nice, just one security comment.

**Comment by nickpell** [comment]
just used this, it's great!

**Pull request merged**